src/core/gestures/MdGestureConfig.ts

```diff
@@ -0,0 +1,21 @@
+import {Injectable} from 'angular2/core';
+import {HammerGestureConfig} from 'angular2/src/platform/browser_common';
+
+/* Adjusts configuration of our gesture library, Hammer. */
+@Injectable()
+export class MdGestureConfig extends HammerGestureConfig {
+  /* List of new event names to add to the gesture support list */
+  events: string[] = ['drag', 'longpress'];
+
+  /*
+  * Overrides default recognizer event names and thresholds.
+  *
+  * More information on these recognizers can be found in Hammer docs:
+  * http://hammerjs.github.io/recognizer-pan/
+  * http://hammerjs.github.io/recognizer-press/
+  * */
+  overrides: {[key: string]: Object} = {
+    'pan': {event: 'drag', threshold: 6},
+    'press': {event: 'longpress', time: 500}
```

Comment:
Can you add any info on where the gesture names (and the `6` / `500`) come from?
(hopefully somewhere in the MD spec)